Multisig/Logic/Models/KeyInfo.swift

```diff
@@ -92,20 +132,63 @@ extension KeyInfo {
 
         if let existing = try context.fetch(fr).first {
             item = existing
+            guard existing.keyType == .device else {
```

Comment:
our app does not allow importing address that is already exists - this function will not be called in this case, anyway

Reply:
update was required for migrateLegacySigningKey